tests/blend.py

```diff
@@ -64,164 +74,151 @@ def colors(request):
+        [[255 / 65535, 0, 0]],
+        [[1, 0, 0]]
+    ], dtype=np.float32)
 
-    result = linear_bgr([{
-        'image': channel_low_med_high,
-        'color': color_white,
-        'min': range_all[0],
-        'max': range_all[1]
-    }])
+    result = make_rgb(f32_low_med_high, color_red)
 
-    np.testing.assert_array_equal(expected, result)
+    np.testing.assert_array_equal(expected[:, :, 0], next(result))
+    np.testing.assert_array_equal(expected[:, :, 1], next(result))
+    np.testing.assert_array_equal(expected[:, :, 2], next(result))
 
 
-def test_color_red(channel_low_med_high, range_all, color_red):
-    '''Blend an image with one channel, testing red color'''
+def test_make_rgb_color_white(f32_low_med_high, range_all, color_white):
+    '''Blend an image with one channel, testing white color'''
 
-    expected = np.uint8([
+    expected = np.array([
         [[0, 0, 0]],
-        [[0, 0, 1]],
-        [[0, 0, 255]]
-    ])
+        [color_white * 255 / 65535],
+        [color_white]
+    ], dtype=np.float32)
 
-    result = linear_bgr([{
-        'image': channel_low_med_high,
-        'color': color_red,
-        'min': range_all[0],
-        'max': range_all[1]
-    }])
+    result = make_rgb(f32_low_med_high, color_white)
 
-    np.testing.assert_array_equal(expected, result)
+    np.testing.assert_array_equal(expected[:, :, 0], next(result))
+    np.testing.assert_array_equal(expected[:, :, 1], next(result))
+    np.testing.assert_array_equal(expected[:, :, 2], next(result))
 
 
-def test_color_khaki(channel_low_med_high, range_all, color_khaki):
-    '''Blend an image with one channel, testing khaki color
-    Colors of any lightness/chroma should map low uint16 input values to 1
+def test_make_rgb_color_khaki(f32_low_med_high, color_khaki):
+    '''Make an image with one channel, testing khaki color
+    Colors of any lightness/chroma should correctly normalize
     '''
 
-    expected = np.uint8([
+    expected = np.array([
         [[0, 0, 0]],
-        [[1, 1, 1]],
-        [[140, 230, 240]]
-    ])
+        [color_khaki * 255 / 65535],
+        [color_khaki]
+    ], dtype=np.float32)
 
-    result = linear_bgr([{
-        'image': channel_low_med_high,
-        'color': color_khaki,
-        'min': range_all[0],
-        'max': range_all[1]
-    }])
+    result = make_rgb(f32_low_med_high, color_khaki)
 
-    np.testing.assert_array_equal(expected, result)
+    np.testing.assert_array_equal(expected[:, :, 0], next(result))
+    np.testing.assert_array_equal(expected[:, :, 1], next(result))
+    np.testing.assert_array_equal(expected[:, :, 2], next(result))
 
 
-def test_color_khaki_range_low(channel_low_med_high, range_low, color_khaki):
+def test_linear_rgb_khaki_low(u16_low_med_high, range_low, color_khaki):
     '''Blend an image with one channel, testing khaki at low range
-    Colors of any lightness/chroma should map inputs above threshhold to 0
+    Colors of any lightness/chroma should set all over max to 1
```

Comment:
Suggest: "Ensure overbright values are clipped to 1"

Reply:
done in `refactor names and revert to version 0.0.1`

Reply:
https://github.com/sorgerlab/minerva-lib-python/pull/17/commits/29ed61d369e2d506ea15686cd45fd49e12580ee1#diff-4438daa1ceea9cdeeef65ab8cfb5075dR211